### PR TITLE
Separate header

### PR DESCRIPTION
**Pull request by jadilorenzo**
Screen Shot 2020-02-26 at 9 21 45 PM

### CODE REVIEW

File: src/components/AboutPage.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 import react from './icon/react.svg'
 import tailwind from './icon/tailwind.svg'
 import postgres from './icon/postgres.png'
 
 function AboutPage() {
   return (
     <div className="h-screen w-screen bg-gray-100">
-      <div className={`header`}>Messages</div>
+      <Header>
+        Resourses
```
Comment (rcdilorenzo) [inline src/components/AboutPage.js]: You've got a typo here.

```suggestion
        Resources
```

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Link} from 'react-router-dom'
+
+export default (props) => {
+  return (
+    <Link to='/'>
+      <div className='header'>
+        <span aria-label='' role='img'>{props.children}</span>
```
Comment (rcdilorenzo) [inline src/components/Header.js]: I'm pretty sure the `role` shouldn't be `img` here. Just leave out that and the `aria-label` for now.

File: src/components/LandingPage.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 import login from './icon/login.svg'
 import menu from './icon/menu.svg'
 import userAdd from './icon/user-add.svg'
 
 function LandingPage() {
   return (
     <div className="h-screen w-screen bg-gray-100">
-      <div className='header'>Messages <Link to='/'>💬</Link></div>
+      <Header>
+        Messgaes
```
Comment (rcdilorenzo) [inline src/components/LandingPage.js]:
```suggestion
        Messages
```
Another typo...

**Comment by jadilorenzo** [comment]
Screen Shot 2020-02-28 at 8 17 41 AM

**Pull request merged**